src/settings/index.ts

```diff
@@ -18,6 +18,8 @@ import defaultMapIcon from "assets/icons/default-map-icon.png";
 import sparkleNavLogo from "assets/icons/sparkle-nav-logo.png";
 import sparkleverseLogo from "assets/images/sparkleverse-logo.png";
 
+export * from "./url-settings";
+export * from "./use-settings-defaults";
```

Comment:
I think all of the defaults should be in the file, I know it's already 600+ lines of code but it might be easier if they are all in one place, at least for now, until we refactor this.

Reply:
easier for what? they're still imported from `/settings`

Reply:
renamed @ f908a1489cfee5b6d593c45708b24e78493e99b5